### ISSUE TITLE
generic GAN question color space collapse

### ISSUE THREAD

**Issue by kbuggenhout** [dvschultz/stylegan2-ada-pytorch#1]
I am wondering if you have any tips, my training ( using transfer learning from FFHQ network) runs fine for a few 1000 Kimgs then starts to show a tendency to go monochromatic, into one hue of a color. my input set is very random, maybe thats one of the problems and non labeled. when I restart the training with the newly trained network, the color comes back, but quality of the images deteriorates and the fid metric rises continously indicating some divergence. visually it seems still ok, but objects become more distorted.

would lowering the learning rate help here, I tried to turn off all adaptive augmentation but still get distortion.
we are achieving our goal to some extent, the end result being objects that at first sight are one thing untill you inspect closer and you see that it may be actually having aspects of two or three objects from the original dataset.

apart from the slow startup, the pytorch seems faster and better behaving than the tensorflow version, but this could be subjective.

thnx

**Comment by dvschultz** [comment]
i recommend starting by turning off the color augmentations `--augpipe='bg'`
